resources/authgear/templates/en/web/__watermark.html

```diff
@@ -1,15 +1,15 @@
 {{ define "__watermark.html" }}
-<div class="flex justify-center mt-5">
   {{ if $.WatermarkEnabled }}
+  <div class="flex justify-center mt-5">
     {{ if $.IsNativePlatform }}
-    <div>
-      <span class="footer-watermark"></span>
-    </div>
-    {{ else }}
-    <a href="https://www.authgear.com/?utm_campaign=uibrand" target="_blank">
-      <span class="footer-watermark"></span>
-    </a>
+      <div>
+        <span class="footer-watermark"></span>
+      </div>
+      {{ else }}
+      <a href="https://www.authgear.com/?utm_campaign=uibrand" target="_blank">
+        <span class="footer-watermark"></span>
+      </a>
+      {{ end }}
     {{ end }}
-  {{ end }}
-</div>
+  </div>
```

Comment:
1. Please correct the indentation.
2. The `</div>` is outside the `if` unexpectedly

Reply:
After we changed the `__watermark.html` layout, will it affect the layout of the normal page?

Reply:
Looks like I removed the commit that fixes other normal pages. Reverted change on watermark template and handled the vertical padding in the notices instead.